src/commands/createNewProject/createNewProject.ts

```diff
@@ -106,3 +110,25 @@ export function getProjectCreator(language: string, functionAppPath: string, act
             return new ScriptProjectCreatorBase(functionAppPath, actionContext.properties);
     }
 }
+
+async function addToFilesExclude(filesToExclude: string | string[], fsPath: string): Promise<void> {
+    const projectConfiguration: WorkspaceConfiguration = workspace.getConfiguration('files', Uri.file(fsPath));
+    // tslint:disable:no-any no-unsafe-any
+    let excludedFiles: any | undefined = projectConfiguration.get('exclude');
+    if (excludedFiles) {
+        for (const key of Object.keys(DefaultFilesExcluded)) {
+            delete excludedFiles[DefaultFilesExcluded[key]];
+        }
+    } else {
+        excludedFiles = {};
+    }
+
+    if (filesToExclude.length > 0) {
+        for (const file of filesToExclude) {
+            excludedFiles[file] = true;
+        }
+    } else {
+        excludedFiles.filesToExclude = true;
```

Comment:
Huh?

Reply:
This is to handle the case where it is a string[] or a string.

Reply:
You're setting it to:
  "files.exclude": {
    filesToExclude: true
  }

??